### PR TITLE
Parser done for Kisalfold.

### PR DESCRIPTION
**Pull request by NyistMilan**

### CODE REVIEW

File: summarization/html_parsers/kisalfold_parser.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        if not lead:
+            lead = next(iter(soup.select('article.left-column > p.lead')), None)
+
+        return self.get_text(lead, '')
+
+    def get_article_text(self, url, soup) -> str:
+        article_text = self.get_text(soup.find('div', id='article_text'))
+
+        if not article_text:
+            article = soup.select('div.enews-article-content > p')
```
Comment (dorinapetra) [inline summarization/html_parsers/kisalfold_parser.py:48]: Are you sure the p tag is enough? What if the article has a list too? (also the same problem below this)

File: summarization/html_parsers/kisalfold_parser.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            article_text = "".join([self.get_text(text, remove_img=True) for text in article])
```
Comment (dorinapetra) [inline summarization/html_parsers/kisalfold_parser.py:49]: I think you should join the paragraphs with new lines (but at least some whitespace). And all the other places below this.

**Review by dorinapetra — CHANGES_REQUESTED** [review]
I have some suggestions but otherwise, it's LGTM.

File: summarization/html_parsers/kisalfold_parser.py (hunk 1 above)
Comment (NyistMilan) [reply summarization/html_parsers/kisalfold_parser.py:48]: Should I generalize and just use `div.enews-article-content`? Every article I've looked at had 1 or 2 `p` tags, but there could be a case when something else is present.

Comment (dorinapetra) [reply summarization/html_parsers/kisalfold_parser.py:48]: Yes, I think that should be enough

Comment (NyistMilan) [reply summarization/html_parsers/kisalfold_parser.py:48]: Found an example that gets parsed wrong:
https://www.kisalfold.hu/orszag-vilag/2014/08/egy-heten-belul-johetnek-az-ujabb-oroszellenes-szankciok

**Review by dorinapetra — APPROVED** [review]

**Pull request merged**